apps/quotes/serializers.py

```diff
@@ -16,28 +16,23 @@ class QuoteSerializer(SearchableModelSerializer):
 
     bite = serpy.MethodField()
     html = serpy.Field()
-    truncated_html = serpy.MethodField()
     has_multiple_attributees = serpy.BoolField()
```

Comment:
Do we not want `hasMultipleAttributees`?

Reply:
Looks like it's not being used in any front-end code.